packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -0,0 +1,56 @@
+import { IDropdownOption, IDropdownProps } from '../../Dropdown';
+import { IIconProps } from '../../Icon';
+
+export interface IComboBox {
+
+}
+
+export interface IComboBoxProps extends IDropdownProps {

```

Comment:
The comboBox doesn't use a dropdown since dropdowns allow you to style the visible collapsed where comboBoxes enforce that the rendering is inside of a `<input>` element which should not allow for an arbitrary render functionality